app/src/main/java/com/android/sample/ui/profile/Profile.kt

```diff
@@ -0,0 +1,153 @@
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.sample.R
+import com.android.sample.model.profile.ProfileData
+import com.android.sample.ui.navigation.BottomNavigationMenu
+import com.android.sample.ui.navigation.LIST_TOP_LEVEL_DESTINATION
+import com.android.sample.ui.navigation.NavigationActions
+import com.android.sample.ui.theme.PrimaryGradientBrush
+import com.android.sample.ui.theme.PrimaryPurple
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ProfileScreen(user: ProfileData, navigationAction: NavigationActions) {
+  Scaffold(
+      modifier = Modifier.testTag("profileScreen"),
+      topBar = {
+        TopAppBar(
+            colors = TopAppBarDefaults.topAppBarColors(titleContentColor = PrimaryPurple),
+            // Username displayed
+            title = {
+              Text(
+                  text = user.username,
+                  fontWeight = FontWeight.Bold,
+                  fontSize = 20.sp,
+                  modifier = Modifier.testTag("titleUsername"))
+            },
+            actions = {
+              // Notification icon
+              IconButton(
+                  modifier = Modifier.testTag("profileScreenNotificationsButton"),
+                  onClick = {
+                    // Handle navigation icon click
+                  }) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.bell_pin_fill),
+                        modifier = Modifier.size(28.dp),
+                        tint = Color.Unspecified,
+                        contentDescription = "Notifications")
```

Comment:
This is good for now, maybe consider using an icon instead of an image for a more cohesive design.